### DISCUSSION TITLE
GPS location

### DISCUSSION THREAD

**Discussion by davidribs** [UmerCodez/SensorServer#20] [Ideas]
Hey just wanted to let you know there's still interest in the ability to capture location data. Thanks for your work.

**Comment by UmerCodez** [comment]
@davidribs GPS will be available in upcoming versions

**Reply by davidribs** [reply]
Not totally sure about how your implementation will work but I've been testing out qpython and collecting precise location data and there seems to be a limit to how precise 
and how often the longitude and latitude data can be updated. Have you encountered any similar problems?

**Reply by UmerCodez** [reply]
> Not totally sure about how your implementation will work but I've been testing out qpython and collecting precise location data and there seems to be a limit to how precise and how often the longitude and latitude data can be updated. Have you encountered any similar problems?

Not sure about qpython but Android API provides [requestLocationUpdates(provider,minTime,MinDistance,listener)](https://developer.android.com/reference/android/location/LocationManager#requestLocationUpdates(java.lang.String,%20long,%20float,%20android.location.LocationListener)) method to register app for location updates where <html>
<body>


minTime |  minimum time interval between location updates in milliseconds
-- | --
minDistance |  minimum distance between location updates in meters


</body>
</html>

**Comment by UmerCodez** [comment]
@davidribs I have updated the code, GPS feature will be available in the next release 6ae18fad83b182542fa35372ce2c7c1123ffd0ce

**Comment by UmerCodez** [comment]
GPS feature is now available. Check latest release.
